base/exports.jl

```diff
@@ -981,6 +982,7 @@ export
     enumerate,
     next,
     start,
+    visit,
```

Comment:
I basically agree with this. So the question is, do we want `for (i, a) in enumerate(A)` or `for (i, a) in Enumerate(A)`? I kind of prefer the lowercase version, and un-export the type.